client/ayon_traypublisher/plugins/create/create_editorial_advanced.py

```diff
@@ -0,0 +1,1341 @@
+            except AttributeError:
+                track_start_frame = 0
+
+            for otio_clip in track.find_clips():
+                if not self._validate_clip_for_processing(otio_clip):
+                    continue
+
+                clip_related_content = clip_content.get(otio_clip.name)
+
+                if ignore_clip_no_content:
+                    if not clip_related_content:
+                        continue
+
+                    if not any(
+                        item
+                        for preset in product_type_presets
+                        for item in clip_related_content
+                        if preset["product_name"] in item["product_name"]
+                    ):
+                        self.log.warning(
+                            f"Clip {otio_clip.name} has no related content."
+                            " Skipping clip."
+                        )
+                        continue
+
+                # convert timeline range to source range
+                self._restore_otio_source_range(otio_clip)
+
+                base_instance_data = self._get_base_instance_data(
+                    otio_clip,
+                    instance_data,
+                    track_start_frame,
+                    folder_entity
+                )
+
+                parenting_data = {
+                    "instance_label": None,
+                    "instance_id": None
+                }
+
+                self._make_shot_product_instance(
+                    otio_clip,
+                    deepcopy(base_instance_data),
+                    parenting_data,
+                )
+
+                abs_clip_folder = os.path.join(
+                    media_folder_path, otio_clip.name).replace("\\", "/")
+
+                for pres_product_data in product_type_presets:
+                    self._make_product_instance(
+                        pres_product_data,
+                        deepcopy(base_instance_data),
+                        parenting_data,
+                        clip_related_content,
+                        abs_clip_folder,
+                    )
+
+    def _include_files_for_processing(
+        self,
+        product_name,
+        files,
+        product_data_base,
+        collecting_items,
+        strict=True,
+    ):
+        """Supporting function for getting clip content.
+
+        Args:
+            product_name (str): product name
+            partial_path (str): clip folder path
+            files (list): list of files in clip folder to collect
+            product_data_base (dict): product data
+            collecting_items (list): list for collecting product data items
+            strict (Optional[bool]): strict mode for filtering files
+        """
+        # compile regex pattern for matching product name
+        pattern_search = re.compile(
+            f".*({re.escape(product_name)}{VARIANTS_PATTERN})"
+        )
+        self.log.warning(f">>> pattern_search: {pattern_search}")
```

Comment:
There is a lot of `self.log.warning`, which I guess is because debug messages did not print for you, but these really should be `self.log.debug`...